reporting_app/__init__.py

```diff
@@ -574,3 +564,51 @@ def species_page(species):
             )
         ]
     )
+
+
+@app.route('/charts/<view_type>')
+@flask_login.login_required
+def plotting_report(view_type):
+    if view_type == 'last_month':
+        time_ago = util.now() - datetime.timedelta(days=30)
+    elif view_type == 'last_3_months':
+        time_ago = util.now() - datetime.timedelta(days=182)
+    elif view_type == 'last_12_months':
+        time_ago = util.now() - datetime.timedelta(days=365)
+    else:
+        fl.abort(404)
+        return None
+
+    return render_template(
+        'charts.html',
+        api_urls=[
```

Comment:
Would be good to add a title here as well

Reply:
Added a title